Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -1170,30 +1182,21 @@ - (void)configurePhotoCell:(WMFPicOfTheDayCollectionViewCell *)cell withImageInf
 
 - (void)configureNewsCell:(WMFNewsCollectionViewCell *)cell withContentGroup:(WMFContentGroup *)contentGroup layoutOnly:(BOOL)layoutOnly {
     cell.layoutMargins = self.readableMargins;
-    NSArray *stories = contentGroup.content;
-    WMFFeedNewsStory *story = [stories firstObject];
+    WMFFeedNewsStory *story = (WMFFeedNewsStory *)contentGroup.contentPreview;
     if ([story isKindOfClass:[WMFFeedNewsStory class]]) {
         [cell configureWithStory:story dataStore:self.userStore theme:self.theme layoutOnly:layoutOnly];
     }
 }
 
 - (void)configureOnThisDayCell:(WMFOnThisDayExploreCollectionViewCell *)cell withContentGroup:(WMFContentGroup *)contentGroup layoutOnly:(BOOL)layoutOnly {
     cell.layoutMargins = self.readableMargins;
-    NSArray *events = contentGroup.content;
-    NSInteger featuredIndex = contentGroup.featuredContentIndex;
-    WMFFeedOnThisDayEvent *event = (WMFFeedOnThisDayEvent *)contentGroup.featuredContentObject;
-
-    if ([event isKindOfClass:[WMFFeedOnThisDayEvent class]]) {
-        WMFFeedOnThisDayEvent *previousEvent = event;
-        NSInteger attempts = 0;
-        while (events.count > featuredIndex + 1 && previousEvent.year == event.year && attempts < 4) {
-            WMFFeedOnThisDayEvent *potentialEvent = events[featuredIndex + 1];
-            if ([potentialEvent isKindOfClass:[WMFFeedOnThisDayEvent class]]) {
-                previousEvent = potentialEvent;
-            }
-            attempts++;
+    NSArray *previewEvents = (NSArray *)contentGroup.contentPreview;
+    if ([previewEvents isKindOfClass:[NSArray class]]) {
+        WMFFeedOnThisDayEvent *event = previewEvents.firstObject;
+        WMFFeedOnThisDayEvent *previousEvent = previewEvents.count > 1 ? previewEvents.lastObject : nil;
```

Comment:
`previousEvent` should be the first event before `event` which doesn't have the same year. Otherwise on the feed card it's possible to see the same year twice. On Oct 2, for example, it should show `1990` as the faded out year at the bottom:
<img width="431" alt="screen shot 2017-10-12 at 3 29 13 pm" src="https://user-images.githubusercontent.com/3143487/31522437-a4f45e3a-af62-11e7-8682-e3d29e5f70b7.png">

I double-checked on develop and it's broken there too, so fix doesn't have to come with this PR, but saw this PR was changing the `previousEvent` logic, so thought I'd mention it. (also double-checked with Carolyn that that's what was wanted).

Reply:
(I'll file ticket if you'd rather fix separately)